client/my-sites/plugins/plugin-install-button-wpcom/index.jsx

```diff
@@ -56,9 +57,11 @@ export const WpcomPluginInstallButton = ( props ) => {
 		}
 	}
 
+	const label = isTransferring ? translate( 'Installing…' ) : translate( 'Install' );
```

Comment:
I'm not seeing this change while the site is transferring

<img width="793" alt="image" src="https://user-images.githubusercontent.com/96462/116509252-e5570180-a8ba-11eb-9b4d-b3f4550edc2a.png">

Reply:
Let me take a look.

Reply:
Fixed